tstrait/trait_model.py

```diff
@@ -122,10 +122,11 @@ class TraitModelExponential(TraitModel):
     ----------
     scale : float
         Scale of the exponential distribution. Must be non-negative.
-    negative : bool, default False
-        Determine if a negative can be simulated from the trait model. If it is set
-        to be True, :math:`1` or :math:`-1` will be randomly multiplied to the
-        simulated effect sizes.
+    random_sign : bool, default False
+        If True, :math:`1` or :math:`-1` will be randomly multiplied to the
+        simulated effect sizes, such that we can simulate constant value effect
```

Comment:
this "constant value" is incorrect right? Can you be a bit more careful about copying and pasting text around please?

Reply:
I'm sorry about that, and I just fixed it now.